pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/genqlient.graphql

```diff
@@ -72,6 +94,19 @@ query getCommitData(
   }
 }
 
+
+# Query cost breakdown (highlights are the only things that cost points)
+# repository: 1
+#   refs: 1 (even though we fetch $branchFrist branches it is a single request)
+#     nodes:
+#       compare: 1 * $branchFirst (this has to be done for each branch)
+#
+# Cost = Math.floor((1 + 1 + $branchFirst) / 100)
```

Comment:
This is not valid latex. I was trying to find good way to convey that we want to round down. It looks like if we want to use valid latex it would be something along the lines of

```suggestion
#    C=\left \lfloor{(1 + 1 + x) / 100}\right \rfloor  \\
#    \begin{aligned}
#        \text{where:} \\
#        x &= \text{branchFirst} \\
#        C &= \text{Cost}
#    \end{aligned}
```

Though these are code comments so I find this harder to understand

Reply:
oh so it's go code? I just wasn't understanding how someone would use that code. My thought is since it's a comment, they wouldn't so would just be regular math.

Reply:
Updated comment to have valid go in https://github.com/liatrio/liatrio-otel-collector/pull/217/commits/667ae24f371eac3575ae76207b5f66cc0efcca09